### PR TITLE
added examples for buck and inductor for v0.5 release

### PR DESCRIPTION
**Pull request by rsnancollas**

### CODE REVIEW

File: support-docs/examples.md [hunk 1]
```diff
@@ -259,4 +261,172 @@ Single Complex Pin Example
...
+    "saturationCurrent": {
+        "siUnit": "amp",
+        "absoluteValue": 6,
+        "unitText": "A",
+        "unitFactor": 1,
+        "valueDefined": true
+    },
+    "saturationCurve": {
+        "title": "DC Bias Curve for abc123",
+        "xLabelUnits": "Amps",
```
Comment (asadate) [inline support-docs/examples.md]: xUnits

File: support-docs/examples.md [hunk 2]
```diff
@@ -259,4 +261,172 @@ Single Complex Pin Example
...
+        "xLabelTitle": "Current",
```
Comment (asadate) [inline support-docs/examples.md]: xlabel

File: support-docs/examples.md [hunk 3]
```diff
@@ -259,4 +261,172 @@ Single Complex Pin Example
...
+        "yLabelUnits": "uH",
```
Comment (asadate) [inline support-docs/examples.md]: yUnits

File: support-docs/examples.md [hunk 4]
```diff
@@ -259,4 +261,172 @@ Single Complex Pin Example
...
+        "yLabelTitle": "Inductance",
```
Comment (asadate) [inline support-docs/examples.md]: yLabel

File: support-docs/examples.md [hunk 5]
```diff
@@ -259,4 +261,172 @@ Single Complex Pin Example
...
+    },
+    "maxLoadCurrent": {
+        "siUnit": "amp",
+        "absoluteValue": 5,
+        "unitText": "A",
+        "unitFactor": 1,
+        "valueDefined": true
+    },
+    "integratedFets": true,
+    "integratedFetProperties": {
```
Comment (asadate) [inline support-docs/examples.md:391]: Do we also want to add ilimHSFET and ilimLSFET properties?

File: support-docs/examples.md [hunk 6]
```diff
@@ -259,4 +261,172 @@ Single Complex Pin Example
...
+            "unitText": "mOhm",
+            "unitFactor": 1E-3,
+            "valueDefined": true
+        },
+    },
+    "package": {
+        "length": 5,
+        "width": 5,
+        "height": 1, 
+        "dimensionUnit": "milimeter"
```
Comment (asadate) [inline support-docs/examples.md]: typo: "millimeter"

File: support-docs/examples.md [hunk 7]
```diff
@@ -259,4 +261,172 @@ Single Complex Pin Example
...
+    },
+    "saturationCurrent": {
+        "siUnit": "amp",
+        "absoluteValue": 6,
+        "unitText": "A",
+        "unitFactor": 1,
+        "valueDefined": true
+    },
+    "saturationCurve": {
+        "title": "DC Bias Curve for abc123",
```
Comment (asadate) [inline support-docs/examples.md:307]: Adding the "numberOfCurves" property may be good here, although this is not mandatory

**Review by asadate — CHANGES_REQUESTED** [review]

File: support-docs/examples.md (hunk 5 above)
Comment (rsnancollas) [reply support-docs/examples.md:391]: We can, I was sort of showing that you could also include a subset if desired since the properties aren't required.

**Comment by rsnancollas** [comment]
edits made

**Review by asadate — APPROVED** [review]

**Pull request merged**